float32[2] center
float32[2] top
float32[2] bottom
sensor_msgs/CompressedImage left_Img
sensor_msgs/CompressedImage right_Img

